lib/domain/model/extensions/platform_file/platform_file_list_extension.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:file_picker/file_picker.dart';
+import 'package:matrix/matrix.dart';
+
+extension PlatformFileListExtension on List<PlatformFile> {
+  List<MatrixFile> toMatrixFileList() {
+    return map(
+      (xFile) => MatrixFile.fromMimeType(
+        bytes: xFile.bytes,
+        name: xFile.name,
+        filePath: '',
+      ),
+    ).toList();
+  }
+}

```

Comment:
```suggestion
extension PlatformFileExtension on PlatformFile {
  MatrixFile toMatrixFile() => MatrixFile.fromMimeType(
    bytes: xFile.bytes,
    name: xFile.name,
    filePath: '',
  );
}

```
then use the map outside the extension.